core/src/test/scala/org/apache/spark/storage/StorageStatusListenerSuite.scala

```diff
@@ -21,6 +21,13 @@ import org.scalatest.FunSuite
 import org.apache.spark.Success
 import org.apache.spark.executor.TaskMetrics
 import org.apache.spark.scheduler._
+import org.apache.spark.SparkConfSuite
+import org.apache.spark.SparkConf
+
+/**
+ * Test the behavior of StorageStatusListener in response to all relevant events.
+ */
+
```

Comment:
spurious extra doc, I don't think you use `SparkConfSuite`, and the import of `SparkConf` is out of order